terraform/environments/data-platform-apps-and-tools/security-groups.tf

```diff
@@ -1,6 +1,6 @@
 module "smtp_vpc_endpoint_security_group" {
   source  = "terraform-aws-modules/security-group/aws"
-  version = "5.1.0"
+  version = "43974e94067251ee464018288aa44862d0adba22"
```

Comment:
Are you sure this will work? Since it's a direct reference to something in the module registry a version tag might not be appropriate (and isn't in line with the other registry modules in this commit)